workitem/workitemtype.go

```diff
@@ -27,32 +28,38 @@ const (
 	SystemIteration           = "system.iteration"
 	SystemArea                = "system.area"
 
-	// base item type with common fields for planner item types like userstory, experience, bug, feature, etc.
-	SystemPlannerItem = "planneritem"
-
-	SystemUserStory        = "userstory"
-	SystemValueProposition = "valueproposition"
-	SystemFundamental      = "fundamental"
-	SystemExperience       = "experience"
-	SystemFeature          = "feature"
-	SystemScenario         = "scenario"
-	SystemBug              = "bug"
-
 	SystemStateOpen       = "open"
 	SystemStateNew        = "new"
 	SystemStateInProgress = "in progress"
 	SystemStateResolved   = "resolved"
 	SystemStateClosed     = "closed"
 )
 
+// Never ever change these UUIDs!!!
```

Comment:
Oh, and I make sure those UUID are also used for any already existing WITs in the DB where we have a match-by-name. See this commit: https://github.com/almighty/almighty-core/commit/15a21ed8e13eb46f5ca05bf425089f296f0646fd